.github/workflows/build.yml

```diff
@@ -84,11 +98,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: recursive
+          ref: ${{ inputs.commit_range_end || github.sha }}
           fetch-depth: 0
+          fetch-tags: true
           
       - name: Get changed components
         run: |
-          $changedComponents = $(./tooling/Get-Changed-Components.ps1 ${{ github.event.before }} ${{ github.event.after }})
+          $changedComponents = $(./tooling/Get-Changed-Components.ps1 ${{ inputs.commit_range_start || github.event.before }} ${{ inputs.commit_range_end || github.event.after }})
```

Comment:
If the default when unspecified is included in the definition at the top, do we need the OR here still within the calls to GetChangedComponents here and below?